client/storage/index.js

```diff
@@ -398,20 +399,18 @@ class Storage {
                         signature
                     };
                     link.validatePledge();
-
-                    // Uncomment after demo for payments
-
-                    // const provider = await link.provider
-                    // const checksumAddress = await this.ctx.web3bridge.toChecksumAddress(`0x${provider.id.split('#')[1]}`)
-                    // await makePayment(checksumAddress, 10) // todo: calculate amount using cost per kb for service provider
-
+                    if (defaultConfig.raiden.switch === 'ON') {
```

Comment:
@Tunji17 - see above about the ctx object. Its possible to access a nodes config like so `this.ctx.config` ... and then any of the params that are in the `defaultConfig`.